tests/setTest.js

```diff
@@ -0,0 +1,64 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Set from '../src/set'
+
+chai.use(chaiChange)
+
+describe('Set', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(Set).to.be.a('function')
+  })
+
+  context('add()', () => {
+    it('adds element to set', () => {
+      const mySet = new Set(['a', 'b', 3, 'zeze'])
+      console.log(mySet)
+
+      expect(() => mySet.add('foo'))
+        .to.alter(() => mySet.size(), { from: 4, to: 5 })
+    })
+  })
+
+  context('isEmpty()', () => {
+    it('checks if set is empty', () => {
+      const mySet = new Set([])
+      let EmptySet = mySet.isEmpty()
+      expect(EmptySet).to.deep.eql(true)
```

Comment:
to.eql is to deep  equal, you don't need to say deep
to.equal isn't deep, but to.eql is.